components/Details/Details.tsx

```diff
@@ -27,15 +29,27 @@ export const Details = ({
     scope: currentScope,
     versions: { current, latest },
   } = useContext(DocsContext);
+  const router = useRouter();
   const scopes = useMemo(() => getScopes(scope), [scope]);
   const [isOpened, setIsOpened] = useState<boolean>(Boolean(opened));
   const isInCurrentScope = scopes.includes(currentScope);
+  const detailsId = title
+    ? title
+        .replace(/[&\/\\#,+()$~%.'":*?<>{}^;\d]/g, "")
+        .replace(/\s/g, "-")
+        .replace(/-$/, "")
+        .toLowerCase()
+    : "title";
```

Comment:
Might be nice to move this into a utils method.

Reply:
I put this conversion into a separate function within this module. At the moment I don't see any way to reuse this conversion in an other places. But if we need to it later, I'll put this function in utils for reuse and will make more universal.
What do you think?